packages/docs/base/color.md

```diff
@@ -1,57 +1,614 @@
 # Color
 
-## Neutral palette
-
-<ul class="sample--color-list">
-  <li class="sample--color is-sample-gray-000"></li>
-  <li class="sample--color is-sample-gray-100"></li>
-  <li class="sample--color is-sample-gray-200"></li>
-  <li class="sample--color is-sample-gray-300"></li>
-  <li class="sample--color is-sample-gray-400"></li>
-  <li class="sample--color is-sample-gray-600"></li>
-  <li class="sample--color is-sample-gray-900"></li>
-</ul>
-
-## Primary palette
-
-### Blue
-<ul class="sample--color-list">
-  <li class="sample--color is-sample-blue-000"></li>
-  <li class="sample--color is-sample-blue-300"></li>
-  <li class="sample--color is-sample-blue-500"></li>
-  <li class="sample--color is-sample-blue-900"></li>
-</ul>
-
-### Green
-<ul class="sample--color-list">
-  <li class="sample--color is-sample-green-000"></li>
-  <li class="sample--color is-sample-green-300"></li>
-  <li class="sample--color is-sample-green-500"></li>
-  <li class="sample--color is-sample-green-900"></li>
-</ul>
-
-### Red
-<ul class="sample--color-list">
-  <li class="sample--color is-sample-red-000"></li>
-  <li class="sample--color is-sample-red-300"></li>
-  <li class="sample--color is-sample-red-500"></li>
-  <li class="sample--color is-sample-red-900"></li>
-</ul>
-
-### Yellow
-<ul class="sample--color-list">
-  <li class="sample--color
-   is-sample-yellow-000"></li>
-  <li class="sample--color is-sample-yellow-300"></li>
-  <li class="sample--color is-sample-yellow-500"></li>
-  <li class="sample--color is-sample-yellow-900"></li>
-</ul>
-
-
-## Secondary palette
-
-<ul class="sample--color-list">
-  <li class="sample--color is-sample-purple-500"></li>
-  <li class="sample--color is-sample-turquoise-500"></li>
-  <li class="sample--color is-sample-orange-500"></li>
-</ul>
+Here's a quick overview of the colors available in Odyssey and some guidelines for their use.
+
+<figure class="ods-table--figure">
```

Comment:
Converting this into a single multi-heading table, per discussion @joejezowski-okta @arnoldsandoval-okta

Reply:
Updated: https://shred.ods.dev/base/color/